spec/linter-stylelint-spec.js

```diff
@@ -263,4 +265,27 @@ describe('The stylelint provider for Linter', () => {
       expect(messages.length).toBe(0);
     });
   });
+
+  describe('will autofix what errors it can', () => {
+    it('saves fixed css if attemptFix is true', async () => {
+      atom.config.set('linter-stylelint.attemptFix', true);
+      atom.config.set('linter-stylelint.disableWhenNoConfig', false);
+      const fixedEditor = await atom.workspace.open(doneFixCSS);
+      const fixedCSS = fixedEditor.getText();
+      const editor = await atom.workspace.open(toFixCSS);
+      await lint(editor);
+      const result = editor.getText();
+      expect(result).toBe(fixedCSS);
+    });
+
+    it('does not saved fixed css if attemptFix is false', async () => {
```

Comment:
"does not attempt fixes if attemptFix is false".

This code isn't saving the file automatically.